setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 setup(name='tap-s3-csv',
-      version='1.3.3',
+      version='1.3.4',
```

Comment:
This also needs a bump of `singer-encodings==0.1.2` down below.

Reply:
Did the change. Now I understand why the changes didn't take into affect

Reply:
Done